### ISSUE TITLE
Move list cards permissions issue

### ISSUE THREAD

**Issue by rrjanbiah** [RestyaPlatform/board#676]
From https://groups.google.com/forum/#!topic/restya/9FzMInZmNiw

I wanted to disable the move list cards for restya members, because I was searching a way to disabling it, when the board is public but not to change anything expect the board owner.

After enabling it, I can't move list cards for owner, for members and for anyone else. What happened?

**Comment by rrjanbiah** [comment]
More note from donat waber from forum:

I moved card Test3 from List "Pendent" to "Anfragen". First it shows correct, but when I reload the board again, then it shows the card, where it was created (see screenshot). In the screenshot you will also see, that there is in the log mentioned, that the card was moved, but it remains on the first list...

I don't know exactly, when it started this behavour. I installed two Apps: Hide ID and Zapier, now I have renamed the app folder, but it still didn't word.
Then I deleted some users which I used for testing. But I don't have any boards left without a owner...

Is it a issue of permission?

**Comment by lmarikannan** [comment]
@rrjanbiah We have tried to reproduce this issue in local but we cannot able to reproduce it. But we found an issue, Drag and Drop functionality is working while Move list cards role is disable for restya members. So we created a ticket for this https://github.com/RestyaPlatform/board/issues/677.

> Then  I deleted some users which I used for testing. But I don't have any boards left without a owner...

This is also we cannot able to reproduce it. Closing this for now.
